### PR TITLE
pfsshell: init at 1.1.0

### PR DESCRIPTION
**Pull request by bignaux**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by bignaux** [comment]
@grahamcofborg build pfsshell

File: pkgs/tools/filesystems/pfsshell/default.nix
Comment (mmahut) [inline pkgs/tools/filesystems/pfsshell/default.nix]: What is the reason to do not ship the stable release?

Comment (bignaux) [reply pkgs/tools/filesystems/pfsshell/default.nix]: Mainly because they' ve changed the buildsystem to meson and i wouldn't rewrite the derivation twice.

File: pkgs/tools/filesystems/pfsshell/default.nix
Comment (sikmir) [inline pkgs/tools/filesystems/pfsshell/default.nix]: What about:
```
postInstall = ''
    install -Dm755 pfsshell -t $out/bin
'';
```

**Comment by bignaux** [comment]
Hello ? have you notice i've fix all your requirement ?

**Review by Valodim — APPROVED** [review]
works as expected :+1:

File: pkgs/tools/filesystems/pfsshell/default.nix
Comment (Valodim) [inline pkgs/tools/filesystems/pfsshell/default.nix]: perhaps add a comment as to why these are necessary?

**Comment by makefu** [comment]
anything blocking this PR?

**Comment by makefu** [comment]
@GrahamcOfBorg build pfsshell

**Comment by Lassulus** [comment]
there seems to be version 1.1.1 maybe we should update before merging

**Pull request closed without merging**
